flow/include/flow/WipedString.h

```diff
@@ -20,16 +20,74 @@
 #ifndef WIPED_STRING_H
 #define WIPED_STRING_H
 #pragma once
+#include <concepts>
 #include <cstring>
 #include <type_traits>
+#include "flow/serialize.h"
 #include "flow/Arena.h"
 #include "flow/FileIdentifier.h"
-#include "flow/serialize.h"
+#include "flow/ObjectSerializerTraits.h"
+
+namespace detail {
+// wraps StringRef for the sole purpose of offering distinct serializable trait
+// we avoid inheritance by choice, to guarantee that no traits are shared with StringRef by unintended implicit
+// conversion
+class WipedStringRef {
+	StringRef* value;
```

Comment:
The name WipedStringRef suggests that it is a ref version of WipedString, but since it is in detail namespace, I understand, the WipedStringRef is intended for internal use for serialization of WipedString. I would suggest to use some more technical name here, like WipedStringSerializationImpl or similar.

Reply:
Let me switch to `WipedStringSerdesWrapper`.